config/crd/bases/kro.run_resourcegraphdefinitions.yaml

```diff
@@ -60,6 +60,50 @@ spec:
             description: ResourceGraphDefinitionSpec defines the desired state of
               ResourceGraphDefinition
             properties:
+              additionalPrinterColumns:
+                description: |-
+                  AdditionalPrinterColumns defines additional printer columns that will
```

Comment:
Need to regenerate the YAML now, as you've changed the comment but it doesn't look like it's propagated to here yet?

Reply:
My bad, I just regenerated the CRD and pushed it